models/media.go

```diff
@@ -910,3 +920,7 @@ func (m *Mediafile) ObtainStreamIds() []string {
 	}
 	return nil
 }
+func (m *Mediafile) ObtainEncryptionKey() []string {
+	fmt.Println(m.encryptionKey)
```

Comment:
```suggestion
	fmt.Println(m.encryptionKey)
```

Remove this line :'(